charger/ocpp/connector.go

```diff
@@ -128,6 +128,23 @@ func (conn *Connector) TransactionID() (int, error) {
 	return conn.txnId, nil
 }
 
+// StatusOCPP returns the unmapped charge point status
```

Comment:
Der Name passt nicht. Das ist ja alles ocpp. Lieber den connector.Status nutzen und die Logik in ocpo.go raus ziehen?

Reply:
Passen tut der Name an sich schon, da diese Status-Bezeichnungen tatsächlich eine native OCPP-Erfindung sind.
Aber ich verstehe was du meinst. Alternativvorschlag?

Wegen dem Locking hatte ich das hier in den Connector gepackt. Nicht gut?

Reply:
Ich mach einen Vorschlag. Eine Methode, Logik verschieben.